File: ios-samples/KiipObjCSample/KiipObjCSample/NativeTableViewCell.m
//
//  NativeTableViewCell.m
//  Sample
//
//  Created by encore on 31/10/18.
//  Copyright © 2018 Kiip, Inc. All rights reserved.
//

#import "NativeTableViewCell.h"

@implementation NativeTableViewCell
@synthesize nativeRewardView;

- (void)awakeFromNib {
    [super awakeFromNib];
    // Initialization code
}

//-(void)prepareForReuse {
//    [super prepareForReuse];
//    nativeRewardView = nil;
//}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

@end

File: ios-samples/KiipSwiftSample/KiipSwiftSample/TableViewCell/NativeTableViewCell.swift
//
//  NativeTableViewCell.swift
//  KiipSwiftSample
//
//  Created by encore on 03/06/19.
//  Copyright © 2019 Kiip. All rights reserved.
//

import UIKit
import KiipSDK

class NativeTableViewCell: UITableViewCell {
    @IBOutlet weak var nativeRewardView: KPNativeRewardView!

    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }

    override func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }
    
}
